src/app/components/campus/campus.component.scss

```diff
@@ -0,0 +1,205 @@
+@import '../../../styles/settings';
+
+/* --------------------------
+  Main Styles
+--------------------------- */
+
+.container {
+	margin-bottom: 6rem;
+	padding: 2rem 0 0 0;
+	text-align: center;
+}
+
+.campus-header {
+	width: 100%;
+	margin-bottom: 3rem;
+	padding: 1rem 4rem;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	color: $sophisticated-white;
+
+	h1 {
+		font-weight: 600;
+	}
+
+	h3 {
+		font-weight: 300;
+	}
+
+	.social-icons a {
+		color: $sophisticated-white;
+		margin: 0 0.5rem;
+	}
+}
+
+.campus-body > div {
+	border-radius: 5px;
+	margin-bottom: 0.5rem;
+}
+
+.subtitle {
+	width: 100%;
+	margin-top: 2rem;
+	margin-bottom: 1rem;
+	padding: 0.75rem;
+
+	h2 {
+		color: $sophisticated-white;
+		text-align: center;
+	}
+}
+
+.more {
+	padding: 1rem;
+	color: $sophisticated-white;
+
+	p {
+		text-align: left;
+		a {
+			color: $sophisticated-white;
+			font-size: 1.5rem;
+		}
+		&#csclink {
+			text-align: left;
+			&a {
+				text-align: center;
+			}
+		}
+	}
+}
+
+.dev-team {
```

Comment:
It looks like this file is copied from the about page, which is totally fine, there's no harm in reusing some styles! However, could you remove the extra styles in this file that you aren't using?